scripts/main.py

```diff
@@ -111,6 +115,7 @@ def print_assistant_thoughts(assistant_reply):
         call_stack = traceback.format_exc()
         print_to_console("Error: \n", Fore.RED, call_stack)
 
+
```

Comment:
?

Reply:
pep8 standards call for 2 lines between methods